### ISSUE TITLE
FirstDirect CSV import

### ISSUE THREAD

**Issue by tesujimath** [egh/ledger-autosync#45]
I'd like to contribute a PR to support import for FirstDirect CSV files.  FirstDirect is a UK bank, and their file format is ludicrously simplistic.

Here's an example:
```
Date,Description,Amount,Balance
04/03/2018,"GROSS INTEREST     TO 03MAR2018",0.04,1599.54
27/02/2018,"MICKEY MOUSE       HAPPY B'DAY",50.00,1599.50
22/02/2018,"MINNIE MOUSE       I LOVE U",500.00,1549.50
04/02/2018,"GROSS INTEREST     TO 03FEB2018",0.03,1049.50
```

The amounts are all GBP, but nothing in the file tells you that.  So I'm thinking we would need a new CLI option for default currency.

The field names are rather generic, but sufficiently different from the existing CSV converters that the CSV subtype selection based on fieldset would still work OK (but may not in future, who knows). 

Does this sound like something you would be prepared to merge?  Should it have a more general name than `FirstDirectConverter`?

**Comment by tesujimath** [comment]
I finally got round to doing this, and by the time I did, I see ledger-autosync had grown a really neat plugin facility.  So I have a plugin, which will be useful to anyone else who banks with First Direct in the UK.

I don't think it's worth including such things in the Python distribution, and it would annoy people who get a false positive on the heading field matches (quite likely I think), but how about an example plugins directory in the source, into which such things could be contributed?
